### DISCUSSION TITLE
Setting cost and resource limits

### DISCUSSION THREAD

**Discussion by cazubris** [Khopa/dcs_liberation#2399] [Q&A] [answered]
Hey all, I was wondering if it is possible to set the cost of a certain aircraft per faction, or perhaps limit the number of items available. For instance, if I'm IRAN and i want to use my existing F14's i would set the cost very low but the total number available to purchase to something low like 8. 
   Another scenario - say Bluefor Modern vs russian PMC. The cost of an f16 to any bluefor modern country friendly to the US would be say 24mil , but since the russian PMC would be unfriendly they would have to buy via third party or pay some cost that is too high, say 99,999,999mil.

**Comment by Starfire13** [comment, marked as answer]
Aircraft costs are not editable in the compiled code.
